app/assets/javascripts/home.js

```diff
@@ -1028,10 +1028,10 @@ $(function() {
       {"id":"gradient-genre-collapsed",  "color":"#b2e2e2", "highlight":"#006d2c"},
       {"id":"gradient-genre-fixed",      "color":"#006d2c", "highlight":"#66c2a4"},
       {"id":"gradient-genre-selected",   "color":"#2ca25f", "highlight":"#edf8fb"},
-      {"id":"gradient-discipline-normal",   "color":"#b30000", "highlight":"#e34a33"}, // reds: #fef0d9, #fdcc8a, #fc8d59, #e34a33, #b30000
-      {"id":"gradient-discipline-collapsed","color":"#fdcc8a", "highlight":"#b30000"},
-      {"id":"gradient-discipline-fixed",    "color":"#b30000", "highlight":"#fc8d59"},
-      {"id":"gradient-discipline-selected", "color":"#e34a33", "highlight":"#fef0d9"},
+      {"id":"gradient-discipline-normal",    "color":"#e2aa00", "highlight":"#efb915"}, // NOW: yellows: #fff6db, #ffdd82, #ffce3d, #efb915, #e2aa00
+      {"id":"gradient-disipline-collapsed",  "color":"#ffdd82", "highlight":"#e2aa00"},
```

Comment:
Is this a typo, or is there a reason you changed the spelling of the selector?

Reply:
That would be a typo, good catch.